zebra-chain/src/notes/sprout.rs

```diff
@@ -1,116 +1,49 @@
-//!
-#![allow(dead_code)]
+//! Sprout notes
 
-use serde::{Deserialize, Serialize};
-use std::{
-    fmt,
-    io::{self},
-};
+#![allow(clippy::unit_arg)]
```

Comment:
Same concern about unit_arg as elsewhere.

Reply:
Ah, this is why, if I remove the allowance, `derive(Arbitrary)` has a sad:

![image](https://user-images.githubusercontent.com/552961/89473084-046f9b00-d750-11ea-8811-d2655b1bb7a0.png)